app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import { 
+    Card, 
+    CardTitle, 
+    CardGroup, 
+    CardImg, 
+    CardBody, 
+    CardSubtitle,} from 'reactstrap'
```

Comment:
It would be best practice to drop the curly brace and everything after to the next line.

Reply:
added the change, thank you